### ISSUE TITLE
Error when running workshop in Windows 10x64

### ISSUE THREAD

**Issue by peper0n** [php-school/workshop-manager#25]
```
d:\cygwin\workshop-manager-master\bin>php workshop-manager

Fatal error: Uncaught DI\NotFoundException: No entry or class found for 'Composer\IO\IOInterface' in D:\cygwin\workshop-manager-master\vendor\php-di\php-di\src\DI\Container.php:119
Stack trace:
#0 D:\cygwin\workshop-manager-master\app\bootstrap.php(37): DI\Container->get('Composer\\IO\\IOI...')
#1 D:\cygwin\workshop-manager-master\bin\workshop-manager(4): require('D:\\cygwin\\works...')
#2 {main}
  thrown in D:\cygwin\workshop-manager-master\vendor\php-di\php-di\src\DI\Container.php on line 119
```

Environment:
```
d:\cygwin\workshop-manager-master\bin>composer --version
Composer version 1.4.1 2017-03-10 09:29:45

d:\cygwin\workshop-manager-master\bin>php --version
PHP 7.0.8 (cli) (built: Jun 21 2016 15:15:15) ( ZTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
```

Same error when running `php workshop-manager.phar verify`

**Comment by barbushin** [comment]
Same error.

**Comment by AydinHassan** [comment]
I can't reproduce this on OSX. @barbushin are you also experiencing the same issue on windows? Unfortunately I don't have a windows box :/. @mikeymike do you have access to a windows machine?

**Comment by mikeymike** [comment]
I don't 😭 ... We haven't got around to updating the docs yet but the preferred but experimental method to run on Windows is using Docker.. the instructions to do it are on [this repo](https://github.com/php-school/docker-phpschool)

@avoska @barbushin are you able to try using Docker ?

**Comment by mikeymike** [comment]
@avoska @barbushin can I just check what terminal you're using to run the workshop manager, we recommend using `ConEmu` .. other terminals tend to either have no ASCII escape sequence support and from our testing ConEmu + Cygwin was the sweet spot with this

**Comment by AydinHassan** [comment]
Looks like there was a problem with the code printing the error message, fixed in ca907e4 .

**Comment by AydinHassan** [comment]
I think this is fixed - feel free to reopen if not :)
